sdk/eventhub/Azure.Messaging.EventHubs/tests/Compatibility/TrackOneRetryPolicyTests.cs

```diff
@@ -60,7 +60,7 @@ public void RetryIsInvokedForRetriableNonEventHubsException()
                 .Setup(policy => policy.CalculateRetryDelay(It.Is<Exception>(value => Object.ReferenceEquals(value, lastException)), It.Is<int>(value => value == retryCount)))
                 .Returns(expectedInterval);
 
-            Assert.That(TrackOne.RetryPolicy.IsRetryableException(lastException), Is.True, "The operation cancelled exception should be considered as retriable by the TrackOne.RetryPolicy.");
+            Assert.That(TrackOne.RetryPolicy.IsRetryableException(lastException), Is.True, "The operation canceled exception should be considered as retriable by the TrackOne.RetryPolicy.");
```

Comment:
Interestingly enough, it seems both spellings are correct.

Reply:
I've always used two "l"s, but given how poorly my typos and spellings have been, I was trusting the dictionary.   :)